config.h.in.cmake

```diff
@@ -263,3 +263,5 @@
 
 
 #cmakedefine USE_POCL_MEMMANAGER
+
+#cmakedefine LLVM_OPAQUE_POINTERS
```

Comment:
Does it always define LLVM_OPAQUE_POINTERS (to 0 or 1)? Just a wild guess what could cause LLVM 14 getting opaque ptrs in to the TCE target.